app/assets/javascripts/govuk_publishing_components/components/cookie-banner.js

```diff
@@ -59,16 +67,16 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     // Hide the cookie banner on the cookie settings page, to avoid circular journeys
     if (newCookieBanner && window.location.pathname === '/help/cookies') {
       this.$module.style.display = 'none'
-    }
-
-    var shouldHaveCookieMessage = (this.$module && window.GOVUK.cookie('seen_cookie_message') !== 'true')
+    } else {
+      var shouldHaveCookieMessage = (this.$module && window.GOVUK.cookie('seen_cookie_message') !== 'true')
 
-    if (shouldHaveCookieMessage) {
-      this.$module.style.display = 'block'
+      if (shouldHaveCookieMessage) {
+        this.$module.style.display = 'block'
```

Comment:
This line is only changed because it's indented, but I wondered what the thinking was behind setting the style like this compared to setting a class that controls `display`? I can't actually think of a good argument for doing it with classes (this way saves lines of code, for example) so this may be a non-point, I just wondered.

Reply:
I don't think there was a particular reason (I think this code was existing already and I followed the pattern). If there were more styles being applied I'd use a class, but not sure it needs it in this case... unless there's a standard pattern for using classes? Happy to change if needed.

Reply:
That's fine then. As long as it's been thought through, and that result seems reasonable.